selfdrive/ui/qt/sidebar.cc

```diff
@@ -43,9 +43,18 @@ Sidebar::Sidebar(QWidget *parent) : QFrame(parent) {
   setFixedWidth(300);
 
   QObject::connect(uiState(), &UIState::uiUpdate, this, &Sidebar::updateState);
+
+  pm = std::make_unique<PubMaster, const std::initializer_list<const char *>>({"userTag"});
 }
 
 void Sidebar::mouseReleaseEvent(QMouseEvent *event) {
+  if (home_btn.contains(event->pos())) {
+    // TODO: maybe only create event while car started?
+    MessageBuilder msg;
+    auto userTag = msg.initEvent().initUserTag();
+    userTag.setId(0);
+    pm->send("userTag", msg);
```

Comment:
@adeebshihadeh Is it okay to create the publisher in ui? Or should some other process be responsible for this?

Reply:
yeah, makes sense to do this in ui